### PR TITLE
Tb/aggrgation

### PR DESCRIPTION
**Pull request by bischtob**
Adding Julia notebook and aggregation tool. Not clean, but useful for a lot of people.

### CODE REVIEW

File: src/py/default_gcm.ipynb [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
.../v37efPNN1m8eDF9+/ZlwYIFTJ06lcOHD99w+TNmzGD79u0sXLiQrKws42fPxRH9aiI6Oppu3brRrVs3ioqKmDZtGt26dePtt98GtBE5J06cyGuvvUa7du0YOXIkUVFRN91V3WAw8Oqrr9K5c2ciIiLYunUr69atu6liTNSuwMBAdu3aRWJiIhkZGZXOxrz33ns89dRT3HHHHcYRGB955BG8vLy4++67iYyMJCEhgaioKF555ZVqj8RXU5cX9atWrSI+Pp7o6GhmzZrFmjVrjG1iYmL45ZdfiIuL45133iEyMrJO4jFpDXf5lbiRqVOnKkDt3r3bOG3AgAHK1ta20gWC11LVRbffffeduvK/feHChcrNza3StG+//VZ169ZNWVlZKWdnZ9WvXz+1bNky4+tz585Vvr6+ytraWg0aNEitWLFCASohIUEpVb2LgAcMGHBTF8wKUR1NPX9mz56tOnTooGxtbZWjo6Pq1q2bmj9/vnEgCiFqQ35+vho3bpyys7NTHh4e6p///KcaMWKEcRCJ7du3q86dOysrK6urcqMq1RlEQiml1q5dq0JCQpS1tbUaMmSI+vrrr41tioqKVGhoqJo4cWKlZY8dO1Z17txZFRcXXzeGa332XD4AhBA1dezYMdWnTx9lY2NjPJ6uPK5ff/115ebmZhzxLjU1VU2YMEF5eHgovV6vAgMD1cSJEyvNcy1XDmCklFLPPffcVd+hxowZo0aPHm18bjAY1LRp01RQUJCytLRUXl5eatSoUSo6OloppY2W+dhjjylnZ2fl5OSkHnvsMTVjxgwVEBBgXMb48ePViBEjbhhPU6ZT6oqOxkIIIYQQ1zBy5Ejc3d3r5L5JQghhCqQLnxBCCCGEEEJUkxRQJuzpp5/G3t6+ysfTTz/d0OEJ0ahJ/ghR+4YNG3bNvJo5c2a9xDBz5sxrxlDToc6FaCihoaHXPI6/+eabhg6v2ZMufCbs3Llz5ObmVvmao6Mjnp6e9RyREKZD8keI2nf27NlrDsbg6upa7eGZb8X58+evORiRjY0Nfn5+dR6DELcqKSmpyuH7QRvx9cr7R4n6JQWUEEIIIYQQQlSTdOETQgghhBBCiGqSAkoIIYQQQgghqkkKKCGEEEIIIYSoJimghBBCCCGEEKKapIASQgghhBBCiGr6f8GTN/6P57VEAAAAAElFTkSuQmCC\n",
```
Comment (blallen) [inline src/py/default_gcm.ipynb:27]: is there a better way to do this? it's quite an ugly chunk of code

**Comment by charleskawczynski** [comment]
I agree regarding the notebook, can't we just convert them to python? I get that notebooks are useful, but they really are not great for version control. Also, I'd be keen to help with Literate examples so that we can generate notebooks from Julia :)

**Comment by bischtob** [comment]
I think we should stay away from Python in the future. It seems a lot of people like notebooks, so we'll need a way to have them.

**Comment by LenkaNovak** [comment]
How about we keep the python notebooks for now? I agree it would be good to use Julia by default, but it would be nice for people to have more options (e.g. if they're not too comfortable with Julia, or if they want to use python-specific packages like Iris). So the dir structure could be src/scripts/julia and src/scripts/py ...

**Pull request merged**